projects/canopy/src/lib/side-nav/side-nav-bar/side-nav-bar.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-side-nav-bar',
+  templateUrl: './side-nav-bar.component.html',
+  styleUrls: ['./side-nav-bar.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LgSideNavBarComponent {
+  @Input() label = 'Nav';
+
+  @HostBinding('class.lg-side-nav-bar') class = true;
+  @HostBinding('attr.role') ariaRole = 'tablist';
+  @HostBinding('attr.aria-label')
+  get ariaLabel() {
+    return this.label ? this.label : 'Nav';
```

Comment:
Will there ever not be a `label` given we provide a default? I wondered if we could...
```ts
@HostBinding('attr.aria-label')
@Input()
label = 'Nav';
``

Reply:
Yep, this works. I've cleaned up the code.